index.html

```diff
@@ -19,14 +19,123 @@ <h1 class="main-text">I'M ZALIHA GLAD TO SEE YOU</h1>
 				<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 			</section>
 			<a href="#" class="connect">LET'S CONNECT</a>
```

Comment:
- [ ] Fix padding/margin in you app.
- [ ] Focus on the font size, especially for the `h1` tag.
- [ ] Add more space between `section` tags.
- [ ]  Add a gray light background for your `body`, but you still maintain the white background for your `header` and `section` tags.
- [ ] [OPTIONAL] Focus on the details to make it more similar to the Figma design you selected.

Figma design | Yours

![image](https://user-images.githubusercontent.com/60164239/129459072-61185162-5a95-4fff-9872-645c68bc9bf1.png)